src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_cleanup.erl

```diff
@@ -30,6 +30,9 @@
 
 -behaviour(gen_fsm).
 
+%% supress gen_fsm deprecation warning
+-compile(nowarn_deprecated_function).
```

Comment:
Would be useful to point to supporting documentation that shows gen_fsm will not be broken in erlang 20 before merging this. Also would be useful to make sure we have tests that will fail in case there are issues with gen_fsm.

Reply:
Deprecations: https://erlang.org/doc/general_info/deprecations.html
Functionality scheduled for removal: https://erlang.org/doc/general_info/scheduled_for_removal.html
gen_fsm listed in current stdlib docs: https://erlang.org/doc/man/gen_fsm.html

Do you want this information placed in comments in the source code?

Reply:
Done.